### ISSUE TITLE
C variable argument list doesn't work in @param (Origin: bugzilla #322353)

### ISSUE THREAD

**Issue by doxygen** [doxygen/doxygen#1924]
status NEW severity *enhancement* in component *build* for *---*
Reported in version *1.4.x* on platform *Other*
Assigned to: Dimitri van Heesch

On 2005-11-24 18:32:56 +0000, Mai-Kee Reiss wrote:

    Version details: 1.4.5
    
    try do doxy this foo:
    
    /*! \brief printf like debug foo
     *
     */
    int my_print( 
       short Level, /*!< \param[in] Level the debug level */
       char const * const fmt, /*!< \param[in] fmt printf-syntaxed string of "C"
    language */
       ...  /*!< \param[in] ... ellipses/variable argument list, see printf() */
       )
    {return 0;}
    
    
    It will result in a usual doc, but the comment ot the "param[in] ..." will
    not
    appear in the document. Reason may be, that "..." is not a valid arg for
    @param.
    Since I do doxy for a lot of C-newbies, I would like to document "..." as a
    normal argument like all others.

On 2006-06-20 06:25:16 +0000, Modem Man wrote:

    Yes, this is bothering me, too!

**Comment by albert-github** [comment]
It is not mandatory to document the `...` argument, but when documentation it inline it was not shown though when documenting it with `\param` it was documented.
The both functions in the example:
```
/*! \brief printf like debug foo
 *
 * \param [in] Level the debug level
 * \param [in] fmt printf-syntaxed string of "C" language
 * \param [in] ... ellipses/variable argument list, see printf()
 */
int my_print0(
   short Level,
   char const * const fmt,
   ...
   )
{return 0;}
/*! \brief printf like debug foo
 *
 */
int my_print(
   short Level,            /*!< [in] the debug level */
   char const * const fmt, /*!< [in] printf-syntaxed string of "C" language */
   ...                     /*!< [in] ellipses/variable argument list, see printf() */
   )
{return 0;}
```
should have the same output but this was not the case.

To correct this I've just pushed a proposed patch, pull request #6844

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of a release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.16.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
